src/resources/action-map/score.js

```diff
@@ -1,86 +1,110 @@
 const assert = require('assert');
 
+const remap = `
+double remap(def value, def map) {
+  for (int i = map.length - 2; i >= 0; i -= 2) {
+    if (map[i] <= value) {
+      return map[i + 1];
+    }
+  }
+  return value;
```

Comment:
```suggestion
  return map.length < 2 ? value : map[1];
```

Reply:
Or if map is always expected to have at least one element maybe
```suggestion
  return map[1];
```